src/components/integrated/OakLessonInfoCard/OakLessonInfoCard.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Meta, StoryObj } from "@storybook/react";
+
+import { OakLessonInfoCard } from "./OakLessonInfoCard";
+
+import { OakFlex, oakIconNames } from "@/components/base";
+
+const controlIconNames = [...oakIconNames].sort();
+
+const meta: Meta<typeof OakLessonInfoCard> = {
+  component: OakLessonInfoCard,
+  tags: ["autodocs"],
+  title: "components/integrated/OakLessonInfoCard",
+  argTypes: {
+    iconName: {
+      options: controlIconNames,
+      control: { type: "select" },
+    },
+  },
+  args: {
+    iconName: "supervision-level",
+    infoCardTitle: "Lesson 1",
+    infoCardDescription:
+      "This is a description This is a description This is a description This is a description This is a description This is a description ",
+    tag: "h1",
```

Comment:
Can we have some realistic content in the default story?

For example:
iconName = `equipment-required`
infoCardTitle = `Equipment`
infoCardDescription = `You will need a copy of the 2017 Frances Lincoln Children's Books edition of ‘Emmeline Pankhurst: Little People, Big Dreams’ by Lisbeth Kaiser and Ana Sanfelippo for this lesson.
tag = `h2`

Ideally we would have more than one example in Storybook to cover this longer example and the shorter examples like the Supervision one.